tardis/atomic/base.py

```diff
@@ -9,9 +9,9 @@ class AtomicData(object):
     def from_hdf(cls, fname):
         return cls(pd.HDFStore(fname, mode='r'))
 
-    def __init__(self, hdf_store, data_types):
+    def __init__(self, hdf_store, data_types=self.all_data_types):
         self.data_types = [data_type()
```

Comment:
I see the confusion. hmm - yes that was definitely not right what I wrote. I think we need to think a little more how we want this object to be initialized. has @mklauser given you an HDF5 file?